dydxV4/dydxV4/_Tracking/dydxCompositeTracking.swift

```diff
@@ -12,23 +12,13 @@ import dydxStateManager
 import Combine
 import Cartera
 import FirebaseAnalytics
-
-enum UserProperty: String {
-    case walletAddress
-    case walletType
-    case network
-    case selectedLocale
-    case dydxAddress
-    case subaccountNumber
-}
+import dydxAnalytics
+import StatsigInjections
+import dydxFormatter
 
 extension TrackingProtocol {
-    fileprivate func set(userId: String?) {
-        self.set(userProperty: .walletAddress, toValue: userId)
-    }
-    
-    func set(userProperty: UserProperty, toValue value: String?) {
-        self.setUserInfo(key: userProperty.rawValue, value: value)
+    func setUserProperty(_ value: Any?, forUserProperty userProperty: UserProperty) {
```

Comment:
Nit: How about renaming it to `setValue(_ value: Any?, forUserProperty: UserProperty)`?  It seems more intuitive.

Reply:
i will in a follow-up PR

Reply:
addressed d8364e2f0a87cf8ea21b5fe932d6301453b2162b